lib/cluster_process.js

```diff
@@ -218,21 +214,18 @@ class ClusterProcess extends EventEmitterEx {
     }
 
     /**
-     * Wrap `fn` to delay it execution to ClusterProcess done initialization.
+     * Resolves when ClusterProcess done initialization.
      * @this {ClusterProcess}
-     * @param {Function} fn
-     * @returns {Function}
+     * @returns {Promise<void>}
      */
-    static whenInitialized(fn){
-        return /** @this {ClusterProcess} */function() {
+    whenInitialized(){
+        return new Promise(resolve => {
```

Comment:
It'd be nice to have single promise here and not create it for every call. Something like
```
get whenInitialized() {
    if (!this._initPromise) {
        this._initPromise = new Promise(resolve => this.once('initialized', resolve));
    }
    return this._initPromise;
}
```